saleor/app/models.py

```diff
@@ -117,6 +117,10 @@ def create(self, app, name="", auth_token=None, **extra_fields):
         app_token.save()
         return app_token, auth_token
 
+    def create_with_token(self, *args, **kwargs):
+        """See https://app.clickup.com/t/2549495/SALEOR-7847 ."""
```

Comment:
IMO, we shouldn't keep internal ticket URLs in code.

Reply:
You can drop the ticket from the code and add it as a review comment. Also, AFAIU adding the `create_with_token` method without updating the `create` is useless.

Reply:
> adding the `create_with_token` method without updating the create is useless.

Not exactly. It lets me use the proper method while waiting for `create` to get fixed without the need to change it afterwards. I say it's convenience.